### DISCUSSION TITLE
setting scroll speed

### DISCUSSION THREAD

**Discussion by ytGrapeFrootZ349** [ppy/osu#23932] [Ideas] [answered]
Maybe insead of dragging the bar to set scoll speed, there should also be another option to input custom ms for scoll speed.
or make dragging the bar more inclusive
I wanted to set my scoll speed to 420ms but i can't due to the bar only going between 441ms and 425ms

**Comment by peppy** [comment, marked as answer]
It is no longer possible to set a specific millisecond value sorry. Please choose 425ms instead.

**Reply by ytGrapeFrootZ349** [reply]
so u won't bring back 420????
(puppy eyes)

**Reply by peppy** [reply]
No, sorry. The change to non-millisecond speed values is a permanent one.
